controllers/grid/settings/genre/form/GenreForm.inc.php

```diff
@@ -71,6 +77,8 @@ function initData($args, $request) {
 				'category' => $genre->getCategory(),
 				'dependent' => $genre->getDependent(),
 				'supplementary' => $genre->getSupplementary(),
+				'key' => $genre->getKey(),
+				'readOnly' => $genre->isDefault(),
```

Comment:
I'd suggest renaming this to `keyReadOnly` or something -- as is, it suggests that the entire form might be read-only.